docs/src/security.md

```diff
@@ -314,7 +305,24 @@ Likewise, Volumes access does not require *privileges* mode or `root` privileges
 Proper permissions must be properly assigned by the Kubernetes platform and/or administrators.
 The PostgreSQL containers run with a read-only root filesystem (i.e. no writable layer).
 
-The operator explicitly sets the required security contexts.
+The operator explicitly sets the required security contexts for all pods and containers in the cluster.
+The [Seccomp Profile](https://kubernetes.io/docs/tutorials/security/seccomp/) to be used for the security 
+context of the PostgreSQL containers can be configured with the `spec.seccompProfile` of the `Cluster` resource.
+If no `seccompProfile` is requested, the `Cluster` containers will use a profile `Type` of `RuntimeDefault`.
```

Comment:
This is forgetting one important point, the OLM case

Reply:
Can you elaborate a little on what you are referring to with the OLM case or point me in the direction of relevant information? I am not so familiar with the details there. From what I have read it is already documented in the security doc that when deployed with OLM the operator can be configured with permissions to manage clusters in all or specific namespaces. I didn't see how this specifically changed how the PostgreSQL security context is modified on PG pods. 

Something I did find was that in OpenShift the PG pods inherit the security context constraints from OpenShift as opposed to be explicitly set on the pods.  I added an additional explanatory note about this to the doc in the PR.

Reply:
https://github.com/cloudnative-pg/cloudnative-pg/blob/main/pkg/specs/pods.go#L426

Reply:
The function to which you are referring is why I added an additional note stating that in the openshift case we inherit the security context based on the security context constraints provided by openshift. Is this sufficient or is more explanation required?